CHANGELOG.md

```diff
@@ -13,6 +13,9 @@ Since v0.0.10, this Changelog is formatted according to the [Common Changelog][c
 - ([#344](https://github.com/demos-europe/demosplan-ui/pull/344)) Add documentation for DpTabs ([@ahmad-demos](https://github.com/@ahmad-demos))
 - ([#333](https://github.com/demos-europe/demosplan-ui/pull/333)) Add "rounded" prop to DpButton, add "arrow-up" and "arrow-down" icons to DpIcon ([@spiess-demos](https://github.com/spiess-demos))
 
+### Internal
```

Comment:
There is no "internal" [change group in Common Changelog](https://common-changelog.org/#24-change-group). I guess it would be ok to list these (after the "Removed" group) as "Changed".

Reply:
8fca69930d8b73581c4f4a1fa883d61c12ad7d34